unable to compile kernel driver for kernel 5.4.1

After upgrading kernel to 5.4.1 I get an error when dkms compiles digimend kernel driver. See attached make.log for more info. OS: Archlinux
[make.log](https://github.com/DIGImend/digimend-kernel-drivers/files/3913499/make.log)

Same for kernel 5.4.2-1 (archlinux) dkms build.

```
Creating symlink /var/lib/dkms/digimend/10/source ->
                 /usr/src/digimend-10

DKMS: add completed.
dkms build digimend/10

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...
make -j16 KERNELRELEASE=5.4.2-arch1-1 KVERSION=5.4.2-arch1-1...(bad exit status: 2)
Error! Bad return status for module build on kernel: 5.4.2-arch1-1 (x86_64)
Consult /var/lib/dkms/digimend/10/build/make.log for more information.
make: *** [Makefile:74: dkms_modules_install] Error 10
```

Make Log in detail:
[make.log](https://github.com/DIGImend/digimend-kernel-drivers/files/3943718/make.log)

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ DKMS_MODULES = $(DKMS_MODULES_NAME)/$(PACKAGE_VERSION)
 DKMS_SOURCE_DIR = $(DESTDIR)/usr/src/$(DKMS_MODULES_NAME)-$(PACKAGE_VERSION)
 
 modules modules_install clean:
-	$(MAKE) -C $(KDIR) SUBDIRS=$(PWD) $@
+	$(MAKE) -C $(KDIR) M=$(PWD) SUBDIRS=$(PWD) $@
 
 depmod_conf_install:
 	install -D -m 0644 depmod.conf $(DEPMOD_CONF)

